setup-jh-dev

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+# Copy and paste this into an executable shell script on the hub.
+## Check out the stsci-announce repo for live work in
+## a JupyterHub notebook session.
+
+set -eux
+
+if [[ $# != 1 ]]; then
+    echo "usage: $0  <branch-to-checkout>"
+    exit 1
+else
+    BRANCH=$1
+fi
+
+git clone https://github.com/jaytmiller/stsci-announce.git
```

Comment:
do we want to use your fork for this?

Reply:
This never worked right really and I think the entire dev process may also be changing;  decided to drop it altogether,  have to figure it out again next time anyway.